aten/src/ATen/templates/DispatchKeyFunctions_inl.h

```diff
@@ -11,14 +12,8 @@
 #ifdef TORCH_ASSERT_ONLY_METHOD_OPERATORS
 #error This change adds a dependency on all pytorch operators, meaning the     \
   file will need to be re-compiled every time an operator is changed or added. \
-  Consider if your change would be better placed in another file, or if a more \
-  specific header might achieve the same goal.
+  Consider including a specific operator from \
+  <ATen/ops/{my_operator}_${dispatch_namespace}_dispatch.h>
 #endif
 
-namespace at {
-namespace ${dispatch_namespace} {
-
-${dispatch_namespaced_declarations}
-
-} // namespace ${dispatch_namespace}
-} // namespace at
+$DispatchKeyFunctions_inl_includes
```

Comment:
Missing {} ?

Reply:
They're actually optional:
https://github.com/pytorch/pytorch/blob/9554ebe44e6e73dc75105d4935d41e626e03299b/tools/codegen/code_template.py#L4

Reply:
Right, but that's the only place where we don't use them? :p 
Not very important.